ansible/playbooks/roles/grafana/tasks/dashboards.yml

```
@@ -23,6 +23,22 @@
       loop: "{{ grafana_online_dashboards }}"
       when: grafana_online_dashboards != []
 
+    - name: Set facts for RabbitMQ dashboard
+      set_fact:
+        rabbitmq_dashboard:
+          - dashboard_id: '10991'
+            datasource: 'Prometheus'
+      when:
+        - specification.rabbitmq_monitoring_enabled is defined
+        - specification.rabbitmq_monitoring_enabled
+
+    - name: Extend external dashboards with RabbitMQ monitoring dashboard
+      set_fact:
+        grafana_external_dashboards: "{{ grafana_external_dashboards + rabbitmq_dashboard }}"
```

```suggestion
        grafana_external_dashboards: "{{ grafana_external_dashboards | union(rabbitmq_dashboard) }}"
```
`union` is for adding when not exists (uniqueness)